config/sources/mvebu-helios4.inc

```diff
@@ -60,4 +60,10 @@ family_tweaks_s()
 	# modify mdadm configuration
 	echo -e "\n # Trigger Fault Led script when an event is detected" >> $SDCARD/etc/mdadm/mdadm.conf
 	echo -e "PROGRAM /usr/sbin/mdadm-fault-led.sh" >> $SDCARD/etc/mdadm/mdadm.conf
+
+	### Ethernet tweaks
+
+	# copy and enable helios4-wol.service
+	cp $SRC/packages/bsp/helios4/helios4-wol.service $SDCARD/usr/lib/systemd/system/
```

Comment:
Shouldn't this be /lib/systemd/system/ ?

Reply:
You right, this is not really correct, I wanted to put in /usr because according to Linux FHS /usr is for vendor specific stuff but there isn't a systemd/system folder under /usr/lib so I guess it's not meant to go there.
I will rise a PR to fix the copy path.